.github/workflows/ci.yml

```diff
@@ -42,6 +42,11 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
+      - name: DP#complex
```

Comment:
If you remove this then we'll still be able to see if the tests work in the DynamicPolynomials tests right ? Then we should do this because I can't merge and release with this

Reply:
Yes, I think so. In DP, I still go to this branch, but once it's merged, I can also remove this customization there.